zrml/prediction-markets/src/lib.rs

```diff
@@ -370,7 +364,7 @@ mod pallet {
                 market.status == MarketStatus::Reported || market.status == MarketStatus::Disputed,
                 Error::<T>::InvalidMarketStatus,
             );
-            let (ids_len, disputes_len) = Self::clear_auto_resolve(&market_id)?;
+            let (ids_len, _) = Self::clear_auto_resolve(&market_id)?;
```

Comment:
Why can we ignore the number of disputes?

Reply:
We currently use Authorized (not simple disputes) and only allow one dispute at most.

Reply:
`simple-disputes` is still active on Battery Station, though, and `MaxDisputes` is `6` there.

Reply:
You are right. Effort analysis: When you come with the strong argument, that `simple-disputes` should be maintained, we should also benchmark all extrinsics with simple-disputes. We decided to benchmark only the `authorized` MDM, though. Yes, I removed the benchmark component here, but just because we only benchmark with one dispute at the moment. I like to stay consistent now and rather ignore this small weight difference for now. Otherwise you need to keep the benchmark component `let d = 1..MaxDIsputes;` inside and do the simple disputes MDM benchmarks for all the code. Just to remove all of this code, when we never use `simple-disputes` in the future again.

Reply:
Sounds ok to me.

My primary concern is how clean this is in terms of software development. In the future, when doing a refactor (like you're doing now with zip-0), the PRs should be structured in a way that this type of decision isn't necessary. The dispute system is in a weird limbo right now. `authorized` doesn't allow multiple disputes, `simple-disputes` still does. Instead, `zrml_authorized` could still allow multiple disputes (even if it's silly) or `simple-disputes` could be deleted.